### DISCUSSION TITLE
how can I help in traslation

### DISCUSSION THREAD

**Discussion by Tato418** [Syslifters/sysreptor#183] [Q&A]
I would like to help translating this to spanish

**Comment by aronmolnar** [comment]
Hi Carlos,
I very much appreciate your offer.  
Do you mean translating the SysReptor user interface (UI)? Do you see the requirement to have it translated?  
(So far, we had no plan to provide other languages than English for the UI).

Thanks.

**Reply by Tato418** [reply]
Not the UI but, the content of the reports eg. vulns, description

**Reply by aronmolnar** [reply]
This would be great.

If you want, you can translate the finding templates (which might make most sense) and our report designs (e.g., Calzone Demo or Margherita Demo). You find all latest versions at
* https://docs.sysreptor.com/assets/demo-templates.tar.gz (Templates)
* https://docs.sysreptor.com/assets/demo-designs.tar.gz (Designs)

...or by visiting the playground via https://docs.sysreptor.com/.

If you translate the templates, we would update the default templates, so that they ship bi-lingual.  
For the designs, there is no built-in language control. But we would provide download links to the translated designs at our docs page.

Thank you for your support! ❤️

**Reply by MWedl** [reply]
We added the source file for demo templates, designs and projects as TOML files to the SysReptor git repo at https://github.com/Syslifters/sysreptor/tree/main/demo_data These TOML files will be packed to tar.gz files in a build pipeline. This improves management of demo data in git (plaintext files instead of binary archives).
